dq-game/src/client/client_controller.py

```diff
@@ -29,12 +32,11 @@ def get_theme_list(self):
 
     def get_client_theme_choices(self):
         # Send instruction to choose themes to clients
-        print("client controller get client theme choices")
         theme_list = self.get_theme_list()
-
         self.renderer.select_themes(theme_list)
-
-        choices = []
-        self.socket.send(choices, "THEME_CHOICE")
+        self.renderer.on("THEMES_CHOICE_DONE", self.send_client_theme_choices)
 
```

Comment:
I think that event listening should be declared in the main scope. Aalso why did you stop injecting the screen handler dependency?

Reply:
True, I moved it to the main